app/src/main/java/co/condorlabs/customcomponents/customcollapsibleview/CollapsibleView.kt

```diff
@@ -470,25 +469,26 @@ class CollapsibleView @JvmOverloads constructor(
     fun setContent(collapsibleContent: View) {
         flContent.addView(collapsibleContent)
 
-        val displayMetrics = DisplayMetrics()
-        (context as? Activity)?.windowManager?.defaultDisplay?.getMetrics(displayMetrics)
-            ?.let {
-                flContent.measure(
-                    displayMetrics.widthPixels,
-                    displayMetrics.heightPixels
-                )
-                contentHeight = flContent.measuredHeight
-            }
+        flContent.viewTreeObserver.addOnGlobalLayoutListener(
```

Comment:
Use null safe operation (`?`) when you are accessing flContent.

Reply:
It's not necessary